bin/pest

```diff
@@ -38,7 +38,7 @@ use Symfony\Component\Console\Output\ConsoleOutput;
             unset($args[$key]);
         }
 
-        if ($value === '--todos') {
+        if (str_contains($value, '--todo')) {
```

Comment:
Perhaps use a strict check to be more specific.
```suggestion
        if (in_array($value, ['--todo', '--todos'], true)) {
```

Just in case someone wants to add a new flag like `--todo-only`